app/services/hyrax/thumbnail_path_service.rb

```diff
@@ -5,7 +5,7 @@ class << self
       # @param [#id] object - to get the thumbnail for
       # @return [String] a path to the thumbnail
       def call(object)
-        return default_image unless object.try(:thumbnail_id)
+        return default_image unless object.try(:thumbnail_id)&.present?
```

Comment:
Catches the case where thumbnail_id is nil and when it is a Valkyrie::ID with an empty string.